lib/web_ui/test/engine/platform_dispatcher_test.dart

```diff
@@ -111,5 +111,43 @@ void testMain() {
       root.style.fontSize = null;
       expect(findBrowserTextScaleFactor(), 1.0);
     });
+
+    test(
+        'calls onTextScaleFactorChanged when the <html> element\'s font-size changes',
+        () async {
+      final html.Element root = html.document.documentElement!;
+      final String oldFontSize = root.style.fontSize;
+      final ui.VoidCallback? oldCallback = ui.PlatformDispatcher.instance.onTextScaleFactorChanged;
+
+      addTearDown(() {
+        root.style.fontSize = oldFontSize;
+        ui.PlatformDispatcher.instance.onTextScaleFactorChanged = oldCallback;
+      });
+
+      root.style.fontSize = '16px';
```

Comment:
Should we instead do `expect(root.style.fontSize, '16px')`? This will ensure that our default test environment expectations are met, and that setting the value to `16px` by itself doesn't trigger mutations.

Reply:
Thanks for the feedback. I agree that an `expect()` would be better here. But, when I run the test, `root.style.fontSize` gives an empty string until I explicitly set it to something else. Is there a more reliable way of getting the default font size so that it doesn't have to be hard-coded? Now that you mention it, the purpose of this line is to check that the current font size is different to '20px', ensuring that `root.style.fontSize = '20px'` triggers a mutation. Would it suffice to use something like `expect(root.style.fontSize, isNot('20px'))`?

Reply:
Ah, I didn't know that the default is empty. Yeah, `isNot('20px')` SGTM.